### PR TITLE
Support Wallaroo Pathrouting

### PR DESCRIPTION
**Pull request by kasuboski**

### CODE REVIEW

File: .devcontainer/Wallaroo-Platform-DS/devcontainer.json [hunk 1]
```diff
@@ -23,25 +24,13 @@
 			]
 		}
 	},
-	"forwardPorts": [ 8080, 8443, 8081],
+	"forwardPorts": [8443],
```
Comment (mnp) [inline .devcontainer/Wallaroo-Platform-DS/devcontainer.json:27]: this isn't 8081 and 8443 used by edge right? unfortunate overload.

File: .devcontainer/Wallaroo-Platform-DS/onCreateCommand.sh [hunk 2]
```diff
@@ -1,27 +1,12 @@
...
-fi
+# workaround for https://github.com/kubernetes-sigs/kind/issues/3696
+# from https://github.com/dvordrova/kind-istio-tempo-otel/issues/1
+sudo tee /etc/docker/daemon.json > /dev/null <<'EOF'
+{
+  "ip6tables": false
+}
+EOF
+sudo pkill -f dockerd
+sudo dockerd --dns 168.63.129.16 &
```
Comment (mnp) [inline .devcontainer/Wallaroo-Platform-DS/onCreateCommand.sh:9]: intentional static ip?

File: .devcontainer/Wallaroo-Platform-DS/devcontainer.json (hunk 1 above)
Comment (kasuboski) [reply .devcontainer/Wallaroo-Platform-DS/devcontainer.json:27]: Correct it's mapped from API lb 443

File: .devcontainer/Wallaroo-Platform-DS/onCreateCommand.sh (hunk 2 above)
Comment (kasuboski) [reply .devcontainer/Wallaroo-Platform-DS/onCreateCommand.sh:9]: Yes it's an azure IP https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16

**Review by mnp — APPROVED** [review]
Seems ok and no risk to product, send it

**Pull request merged**